config.h.example

```diff
@@ -1,24 +0,0 @@
-//
```

Comment:
please leave this .h.example file in the project so that others can get started and know what's expected of a config.h file